src/ansys/pytwin/evaluate/twin_model.py

```diff
@@ -1804,6 +1853,87 @@ def generate_points(self, rom_name: str, on_disk: bool = True, named_selection:
             msg += f"\n{str(e)}."
             self._raise_error(msg)
 
+    def project_tbrom_on_mesh(self, rom_name: str, mesh: pv.DataSet, interpolate: bool, named_selection: str = None):
+        """
+        Project the field ROM data onto a mesh
+
+        Parameters
+        ----------
+        rom_name : str
+            Name of the TBROM considered to project the results.
+        mesh: pyvista.DataSet
+            PyVista DataSet object of the targeted mesh.
+        interpolate: bool
+            Flag to indicate whether the point cloud data are interpolated (True) or not (False) on the targeted mesh.
+            Interpolation is recommended when point cloud data and mesh data are not ordered in the same way, and when
+            the target mesh is different from the one used to generate the ROM. Interpolation is automatically enforced
+            if the target mesh size (i.e. number of cells/points) is different from the point cloud size.
+        named_selection: str (optional)
+            Named selection on which the mesh projection has to be performed. The default is ``None``, in which case the
```

Comment:
I would reformulate `Named selection on which the mesh projection has to be performed.` by `Named selection from the ROM (i.e. subset of ROM points cloud) that will be projected on the targeted mesh.`

Reply:
done